### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by BLOCKSTORMPE** [DarkWav/SAC#66]
Please add that i can set a command which run when a player was detected a specified Times.
Like:
**1: say hi
2: say LOL
3: ban {PLAYER} Hacking**

**Comment by DarkWav** [comment]
Thanks for your feature request! You can already run certain commands after a player has been kicked a certain amount of times. See the config for this feature. I will maybe add special commands for detection,
but i will focus on the real "bugs" in the software first. (e.g. AntiFastBow kicking for using snowballs.)

**Comment by BLOCKSTORMPE** [comment]
Yes and another bug is that the ac often detectes speed hack when there isnt any speed hacker :/

**Comment by DarkWav** [comment]
Thanks for bug reporting!
Could you describe the bug a bit more?
Does this happen to everyone or only to some people? Does this happen in special
Situations? Is there any lag on your server?

**Comment by DarkWav** [comment]
Hopefully fixed now. If not, please reply.

**Comment by DarkWav** [comment]
Im not able to implement the feature request until I implemented a proper violation system.
State changed to waiting

**Comment by Reflyansyah** [comment]
Great
